### PR TITLE
fix issue related to same token transfer with weird callData on nxtp

### PR DESCRIPTION
**Pull request by 8d526**
#### TL;DR
Add no callData or callTo for token transfer thru NXTP.

#### Bug Description
reported by our community members: https://connextscan.io/tx/0xc06de94fb70a6fdaf82a00447d5ec526545fd5ffdca3afe2fc6c59e1ba35b937

He is trying to transfer 5 USDT from polygon to BSC without any cross-chain token swap. However, the callData/CallTo is not empty. NXTP invokes FulfillInterpreter thru this tx: https://bscscan.com/tx/0x8960c4541d6606ef02c85e1c92085583b61f714d56af5859245768f9a63396a4 and looks like our contract failed (simply because there nothing to swap)

In order to avoid this happen again, for same token transfer, we will not include callTo or callData when getting quote. 

#### Test
https://connextscan.io/tx/0xc09f08382c28737681038cb4e517c26b872a0ad103d384299fc69b2c85135fd7
Sent 1 usdt from polygon to BSC

### CODE REVIEW

**Comment by chillvybz** [comment]
Which test cases were tested in this?
Was token transfer AND cross-chain tested? on which chains?

**Comment by 8d526** [comment]
yes I sent 1 usdt from polygon to BSC (cross-chain transfer) 

 https://connextscan.io/tx/0xc09f08382c28737681038cb4e517c26b872a0ad103d384299fc69b2c85135fd7
